package.json

```diff
@@ -107,6 +107,12 @@
             "scope": "window",
             "pattern": "^(|(https?://)api.*.(snyk|snykgov).io)$"
           },
+          "snyk.advanced.useTokenAuthentication": {
+            "type": "boolean",
+            "markdownDescription": "Use token authentication. It is recommended to keep this turned off, as the default OAuth2 authentication is more secure.",
+            "scope": "window",
+            "default": false
```

Comment:
Double-checking, but once https://github.com/snyk/snyk-ls/pull/544 is merged then customers updating to the latest VSCode will need to force upgrade their `snyk-ls` (since it updates every 4 days) right?

Reply:
It will automatically update it, if we increase the protocol version. Probably, we should do that, right?